### PR TITLE
Ternary alignment: fix another nested ternary false error

### PR DESCRIPTION
**Pull request by andyroo2000**
Fixes adobe-community/issues#11113

### CODE REVIEW

**Comment by andyroo2000** [comment]
@mikesherov @ingluisjimenez

File: Behance/Sniffs/ControlStructures/TernarySniff.php [hunk 1]
```diff
@@ -166,8 +191,25 @@ private function _isPrecededByWhitespace( $index ) {
...
+   *
+   * @return bool
+   */
+  private function _isPrecededByWhitespaceOrDesiredToken( $index ) {
+
+    if ( !isset( $this->_tokens[ $index - 1 ] ) ) {
+      return false;
+    }
+
+    return ( $this->_tokens[ $index - 1 ]['code'] === T_WHITESPACE ) ||
```
Comment (bmax) [inline Behance/Sniffs/ControlStructures/TernarySniff.php]: maybe set $index - 1 to a temp var make it a bit easier to read?

Comment (bmax) [inline Behance/Sniffs/ControlStructures/TernarySniff.php]: or pass in `$index - 1`?

**Comment by bmax** [comment]
👍

**Comment by mikesherov** [comment]
Sorry I got to this late. @bmax beat me to it. I think there is a way in phpcs to get the matching opening parenthesis from a closing parenthesis. I think it would make this a bit simpler of a rule.

Let me double check....

**Comment by bmax** [comment]
on the stack pointer for a closing/opening parenthesis there should be these indexes ```
        'parenthesis_opener',
        'parenthesis_closer',
        'scope_opener'
  ```

**Comment by mikesherov** [comment]
Right, exactly @bmax. Does that help @andyroo2000 ?

**Comment by andyroo2000** [comment]
@bmax @mikesherov 
Mind officially blown... I'll update later.
mind-blown

**Pull request merged**